SU2_CFD/src/solvers/CTurbSSTSolver.cpp

```diff
@@ -365,12 +365,25 @@ void CTurbSSTSolver::BC_HeatFlux_Wall(CGeometry *geometry, CSolver **solver_cont
   tie(WallType, Roughness_Height) = config->GetWallRoughnessProperties(Marker_Tag);
   if (WallType == ROUGH ) rough_wall = true;
 
+
+
+  /*--- Evaluate nu tilde at the closest point to the surface using the wall functions. ---*/
+
+  if (config->GetWall_Functions()) {
+    SU2_OMP_MASTER
+    SetTurbVars_WF(geometry, solver_container, config, val_marker);
+    SU2_OMP_BARRIER
```

Comment:
Next time you merge develop you will get some failed compilations with errors about "end macros", for example in this case you will need END_SU2_OMP_MASTER before the barrier.